src/rateEngine/__tests__/rates/calcTOU_EV_7_costs.test.ts

```diff
@@ -0,0 +1,29 @@
+import LoadProfile from "../../LoadProfile";
+import RateCalculator from "../../RateCalculator";
+import { RateCalculatorInterface } from "../../types/index";
+import { RATES } from "./data/RATES";
+
+const dailyLoadProfile = [
+  42.0, 42.0, 42.0, 42.0, 42.0, 30.0, 30.0, 30.0, 30.0, 30.0, 30.0, 30.0, 30.0, 18.0, 18.0, 18.0, 0.0, 0.0, 20.0, 0.0,
+  0.0, 42.0, 42.0, 42.0,
+];
```

Comment:
is this data that we can share publicly outside of an SCE context? (we removed the tests that used the PG&E mocks because they had data we didn't want to share publicly)